packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -52,6 +52,7 @@
 									:placeholder="mappingDropdownPlaceholder"
 									v-model="knobs.timestampColName"
 									:options="datasetColumns?.map((ele) => ele.name)"
+									@change="updateTimeline()"
```

Comment:
Any reason to not call the existing `updateState` and update it to include the `timestampColName`?